global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "2.1.100",
+    "version": "2.1.600",
```

Comment:
I know that 2.1.200 and 2.1.300 have a bug that fails CI like this:

```
========================================
DotnetTest-netcoreapp
========================================
Test run for D:\a\1\s\src\NUnitTestAdapterTests\bin\Release\NUnit.VisualStudio.TestAdapter.Tests.dll(.NETCoreApp,Version=v0.0)
Microsoft (R) Test Execution Command Line Tool Version 15.7.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Cake.exe : The test source file 
"D:\a\1\s\src\NUnitTestAdapterTests\bin\Release\NUnit.VisualStudio.TestAdapter.Tests.dll" provided was not found.
```

2.1.802 (the current one) is too new for AppVeyor. I tried 2.1.600 and it worked. I didn't try 2.1.400 or 2.1.500 because this has taken way too much time and I'm pretty sure no one will care.